py/kubeflow/testing/cleanup_ci.py

```diff
@@ -1088,6 +1103,19 @@ def main():
 
   add_deployments_args(parser_deployments)
   parser_deployments.set_defaults(func=cleanup_deployments)
+
+  ######################################################
+  # Parser for clusters
+  parser_clusters = subparsers.add_parser(
+    "clusters", help="Cleanup clusters")
+
+  parser_clusters.add_argument(
```

Comment:
we have this already?
https://github.com/kubeflow/testing/blob/master/py/kubeflow/testing/cleanup_ci.py#L970

Reply:
That's adding subparser commands and is only for the deployments command. Here we are are doing it for the clusters command.